lib/galaxy/datatypes/converters/ref_to_seq_taxonomy_converter.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+convert a ref.taxonommy file to a seq.taxonomy file
+Usage:
+%python ref_to_seq_taxonomy_converter.py <ref.taxonom> <seq.taxonomy>
+"""
+
+import sys
+import re
+
+assert sys.version_info[:2] >= (2, 4)
+
+
+def stop_err(msg):
+    sys.stderr.write("%s" % msg)
+    sys.exit()
+
+
+def __main__():
+    infile_name = sys.argv[1]
+    outfile = open(sys.argv[2], 'w')
+    for i, line in enumerate(file(infile_name)):
+        line = line.rstrip()
+        if not line or line.startswith('#'):
```

Comment:
What's your opinion on:
`if line and not line.startswith('#'):` and increase indentation of the following 3 lines